uint8 status

#坐标系都是输入的点云的坐标系，也就是相对于“map”系的坐标
float32 min_z
geometry_msgs/Point32 bottom_point
sensor_msgs/PointCloud2 points
